shard.yml

```diff
@@ -18,7 +18,8 @@ targets:
 dependencies:
   redis:
     github: jgaskins/redis
-    version: ~> 0.6.1
+    # version: ~> 0.6.1
+    branch: master
```

Comment:
Should this go back to `~> 0.6.1`? Or at least a specific commit?

Reply:
good catch. i'll make sure this gets a commit or a release tag before merge.

Reply:
it's [now at 0.7.0](https://github.com/jgaskins/redis/releases/tag/v0.7.0)!